targets/TARGET_NORDIC/TARGET_NRF5_SDK13/us_ticker.c

```diff
@@ -284,9 +310,7 @@ static uint32_t previous_tick_cc_value = 0;
  */
 MBED_WEAK uint32_t const os_trv;
 MBED_WEAK uint32_t const os_clockrate;
-MBED_WEAK void OS_Tick_Handler()
-{
-}
+MBED_WEAK void OS_Tick_Handler() { }
```

Comment:
This was fine. The opening brace of each function should be placed in the next line. So this change is undesirable.
The only thing that could be corrected here is missing "void" between parentheses. This is C code, so empty parameters list should be stated with "void".

A few similar cases can be found below.